src/infra/SearchServiceImpl.ts

```diff
@@ -34,7 +34,7 @@ interface UniversalSearchQueryParams extends QueryParams {
   queryTrigger?: QueryTrigger,
   context?: string;
   referrerPageUrl?: string,
-  source?: QuerySource
+  source?: QuerySource | string
```

Comment:
A minor thing, but could we add `SUPPORT_SEARCH` to the `QuerySource` enum? I know it's somewhat irrelevant now that any old string will work. But, I'd like `QuerySource` to list all the "official" or "out of the box" sources so to speak.

Reply:
I asked Rose about this and she felt that 'SUPPORT_SEARCH' is a custom solution thing, so she didn't want it added to the enum